providers/ethereum-provider/test/index.spec.ts

```diff
@@ -48,7 +48,7 @@ describe("EthereumProvider", function () {
       qrModalOptions: {
         themeMode: "dark",
         themeVariables: {
-          "--w3m-z-index": "99",
+          "--wcm-z-index": "99",
```

Comment:
this was left over from when we updated the wcm types, resolving it so ts is satisfied